### ISSUE TITLE
Support for APM Per Request Policy

### ISSUE THREAD

**Issue by mklymslt** [F5Networks/f5-appsvcs-extension#124]
##### ISSUE TYPE

- Feature Idea

##### AS3 BUILD/ VERSION
3.7+
```

```

##### BIGIP VERSION
version 13.1.1

```


##### SUMMARY
while we can attach a per session apm policy to a VS with 

"policyIAM": {
"bigip": "/Common/my-APM-policy"
}

we are not able to attach a per request apm policy to a virtual server and need this functionality



##### EXPECTED RESULTS

have the ability to reference and existing per request apm policy on a virtual server

**Comment by brian-bkt** [comment]
Are there any updates to AT-688? I am in need of this Per-Request-Policy via AS3 Declaration functionality

**Comment by dstokesf5** [comment]
The feature is in development and will likely be available for the 3.23.0 release.

**Comment by cabral1038** [comment]
What is the ETA for the 3.23.0 release?

**Comment by dstokesf5** [comment]
The 3.23.0 release is tentatively planned for mid-October.

**Comment by dstokesf5** [comment]
This was resolved in the 3.23.0 release.
